### DISCUSSION TITLE
React cannot listen to events, such as load, progress, model-visibility, etc.

### DISCUSSION THREAD

**Discussion by liuyang0623** [google/model-viewer#4218] [Q&A]
### Description

I use model-viewer in react to listen to model loading events and customize some loading animations, but I can't listen to load events or model-visibility events. If possible, I want to get the animation list of the model directly from these events

#### Live Demo
<model-viewer
          class="model-view"
          src={file}
          camera-orbit="180deg 90deg 20m"
          max-field-of-view="80deg"
          min-field-of-view="40deg"
          max-camera-orbit="auto 100deg auto"
          interaction-prompt="none"
          camera-controls=""
          orientation="0deg 0deg 145deg"
          autoplay
          shadow-intensity="3"
          exposure=".4"
          ar-status="not-presenting"
          model-visibility={loadModel}
        />

### Version

- model-viewer: "^3.1.0"

### Browser Affected

- [ ] Chrome, version: 版本 111.0.5563.110（正式版本） (x86_64)

**Comment by elalish** [comment]
Why can't you listen to our events? https://modelviewer.dev/docs/index.html#entrydocs-loading-events-modelVisibility Also: https://modelviewer.dev/docs/index.html#entrydocs-animation-properties-availableAnimations
